docs/ko-KR/TIPS.md

```diff
@@ -0,0 +1,20 @@
+<table>
+    <tr>
+        <!-- Do not translate this table -->
+        <td> Read these guidelines in </td>
+        <td><a href="/TIPS.md">English</a></td>
+        <td><a href="/docs/pt-BR/Dicas.md">Português</a></td>
+        <td><a href="/docs/es-ES/TIPS.md">Español</a></td>
+        <td><a href="/docs/ko-KR/TIPS.md">한국어</a></td>
```

Comment:
Could you please add the link to the French translation so at least there are fewer disconnections between the translation's navigation.